pkg/scheduler/api/job_info.go

```diff
@@ -239,24 +239,24 @@ func (ji *JobInfo) SetPodGroup(pg *PodGroup) {
 	ji.CreationTimestamp = pg.GetCreationTimestamp()
 
 	var err error
-	ji.WaitingTime, err = GetWaitingTime(pg)
+	ji.WaitingTime, err = ji.GetWaitingTime(pg)
 	if err != nil {
 		klog.Warningf("Error occurs in parsing waiting time for job <%s/%s>, err: %s.",
 			pg.Namespace, pg.Name, err.Error())
 		ji.WaitingTime = nil
 	}
 
-	ji.Preemptable = GetJobPreemptable(pg)
-	ji.RevocableZone = GetJobRevocableZone(pg)
-	ji.Budget = GetBudget(pg)
+	ji.Preemptable = ji.GetJobPreemptable(pg)
+	ji.RevocableZone = ji.GetJobRevocableZone(pg)
+	ji.Budget = ji.GetBudget(pg)
```

Comment:
If it's not used by others, let's make them as private member funcs.

Reply:
sure